galaxy_api/pulp/galaxy.py

```diff
@@ -0,0 +1,90 @@
+# Copyright 2019 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Pulp Ansible API client."""
+
+import urllib.parse as urlparse
+import requests
+from requests.structures import CaseInsensitiveDict
+
+
+class ApiClient:
+    """Pulp Ansible Galaxy API v3 client."""
+
+    prefix = 'pulp_ansible/galaxy/{_distro}/api/v3/'
```

Comment:
The current galaxy-dev deployment has the pulp api accessible under `/pulp/api/v3/`, will that now be found under this prefix?

Reply:
Nope, these are different APIs.

This is public galaxy API, which is used by mazer and which pulp has mimicked before. Now implementation of v3 public is on the Pulp side and galaxy proxies those endpoints.